drv/transceivers-server/src/udp.rs

```diff
@@ -325,16 +335,24 @@ impl ServerImpl {
             HostRequest::ExtendedStatus(modules) => {
                 ringbuf_entry!(Trace::ExtendedStatus(modules));
                 let mask = LogicalPortMask::from(modules);
-                let (num_status_bytes, result) =
-                    self.get_extended_status(mask, out);
+                let (data_len, result) = if self
+                    .front_io_board_present
+                    .unwrap_or(false)
+                {
+                    self.get_extended_status(mask, out)
+                } else {
+                    (
+                        0,
+                        ModuleResultNoFailure::new(LogicalPortMask(0), mask)
+                            .unwrap(),
```

Comment:
similarly, this probably should be `unwrap_lite`?
```suggestion
                            .unwrap_lite(),
```